SwanContents/swancontents/swanclassic/handlers/projects.py

```diff
@@ -32,23 +38,31 @@ def generate_page_title(self, path):
             return 'My Projects'
 
     @web.authenticated
+    @gen.coroutine
     def get(self, path=''):
         """ Get handler to cope with Projects as root folder """
         path = path.strip('/')
         cm = self.contents_manager
 
         swan_path = url_path_join('SWAN_projects', path)
 
-        if cm.dir_exists(path=swan_path):
-            if cm.is_hidden(swan_path):
+        file_exists = False
```

Comment:
Is this change of logic related to migrating to jupyter_server? What's the motivation for it?

Reply:
This was part of the code changes with CS3 (I was planning to expose the full EOS namespace). I removed it for now, will come later with a different PR.